ui/main/src/app/views/customCardList/resultTable/ResultTableExport.ts

```
@@ -73,7 +73,11 @@ export class ResultTableExport {
         });
     }
 
-    private addResponseFieldsToRow(responseFieldName: string, responses: {color: string; name: string}[], row: {}) {
+    private addResponseFieldsToRow(
+        responseFieldName: string,
+        responses: {color: string; entityName: string}[],
+        row: {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)